pkg/cmd/openshift-controller-manager/controller/build.go

```diff
@@ -57,7 +57,7 @@ func RunBuildController(ctx ControllerContext) (bool, error) {
 			Codec: annotationCodec,
 		},
 		SourceBuildStrategy: &buildstrategy.SourceBuildStrategy{
-			Image: imageTemplate.ExpandOrDie("sti-builder"),
+			Image: imageTemplate.ExpandOrDie("docker-builder"),
```

Comment:
It's the same image today with two different entrypoints.  We're collapsing to one entrypoint.  We shouldn't have even had two entry points, because they look at the build API object and already have to know what type to run.